src/client.js

```diff
@@ -4,22 +4,17 @@ const AccessProvider = require('./accessProvider');
 const RequestHelper = require('./requestHelper');
 
 class Trustpilot {
-  constructor(config) {
-    this.apiKey = config.apiKey;
-    this.baseUrl = config.baseUrl || 'https://api.trustpilot.com';
-    this.secret = config.secret || '';
-    this.username = config.username;
-    this.password = config.password;
-    this.tokenRequest = config.tokenRequest;
+  constructor({ apiKey, baseUrl = 'https://api.trustpilot.com', secret = '', username, password, tokenRequest, accessToken }) {
```

Comment:
The params will no longer be available on the instantiated object, right? If so, then this is a breaking change. And the question is then, if we want to do that.